dataproc/snippets/quickstart/quickstart_test.py

```diff
@@ -59,6 +62,24 @@ def create_bucket(bucket_name):
     return storage_client.create_bucket(bucket_name)
 
 
+@backoff.on_exception(backoff.expo, ServiceUnavailable, max_tries=5)
+def create_subnet():
+    instance_client = compute_v1.SubnetworksClient()
+    try:
+        instance_client.insert(
+            project=PROJECT_ID,
+            region=REGION,
+            subnetwork_resource=compute_v1.Subnetwork(
+                name=SUBNET,
+                network="global/networks/default",
+                ip_cidr_range="172.16.0.0/12",
```

Comment:
This is a big range. How can you find a range that doesn't conflict with other subnetworks?

Reply:
If you have another range, I am happy to try it. I didn't know what might be necessary for this test.